content/kotlin/concepts/datetime/terms/DayOfWeek/DayOfWeek.md

```diff
@@ -0,0 +1,26 @@
+---
+Title: 'DayOfWeek'
+Description: 'The DayOfWeek method will return the '
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Method'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **DayOfWeek** method is used to return the DayOfWeek instance for the week day number, starting with 1 as Monday. 
+## Syntax
```

Comment:
```suggestion

## Syntax

```
We have style guide to leave a white space after heading sub-heading and if a paragraph ends.